docker-compose.yml

```
@@ -0,0 +1,21 @@
+version: '3'
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_USER: root
+      MYSQL_ROOT_PASSWORD: 
```

password空にするなら下記の環境変数追加
```suggestion
      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
```